src/pages/home/_trade-types-mobile.js

```diff
@@ -126,20 +125,27 @@ const TradeTypesMobile = () => {
             overflow: 'hidden',
         },
         slide_style: {
-            minWidth: '24rem',
+            minWidth: '216px',
             borderRadius: '8px',
-            margin: '10px',
+            margin: '10px 16px 48px 0px',
```

Comment:
```suggestion
            margin: '10px 16px 48px 0',
```
`0` doesn't take a unit

Reply:
My bad, I'll commit your suggestion. Thanks~